Tests/Unit/MSFT_xFirewall.Tests.ps1

```diff
@@ -149,6 +149,7 @@ try
                 }
             }
             Context 'Ensure is Present and the Firewall is Absent' {
+                Mock Get-FirewallRule
```

Comment:
I'd be interested in seeing the code you used to test everything it might be interesting if we can get it to go fast enough to do a pass on everything (occassionally).